fv3config/caching.py

```diff
@@ -8,6 +8,15 @@
     os.makedirs(USER_CACHE_DIR, exist_ok=True)
 CACHE_PREFIX = "fv3config-cache"
 
+CACHE_REMOTE_FILES = True
+
+
+def do_remote_caching(flag: bool):
```

Comment:
docstring? short one would be helpful (and maybe a good place to state that default is True?)

Reply:
Added.